fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -20,6 +22,25 @@ def start_pips(argList):
         str(blockID) + " " + str(start) + " " + str(stop),
         shell=True)
 
+    uphill_wires = "wires/uphill_wires_{}.txt".format(blockID)
```

Comment:
Can you explain the reasoning behind this?

Reply:
Here there is a disk utilization optimization only. The fact is that, for big parts, this fuzzer leaves a lot of temporary data. With this change, each process spawned by the pool, updates the final files and deletes the intermediate files, saving disk space. 

Mainly it is done to be able to run this fuzzer locally if the machine does not have enough disk space

Reply:
Only, add a comment to that effect.

Reply:
Done